From b7bcdb9837e802b28e34eb0d69070f54d169f6e8 Mon Sep 17 00:00:00 2001
From: eos <eos@local>
Date: Sun, 7 Jun 2026 08:52:15 +0200
Subject: [PATCH] E-OS: brand login prompt (redox login: -> eos login:)

---
 src/bin/login.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bin/login.rs b/src/bin/login.rs
index 08e178c..cb7c4f9 100644
--- a/src/bin/login.rs
+++ b/src/bin/login.rs
@@ -135,7 +135,7 @@ pub fn main() {
     loop {
         let user = liner::Context::new()
             .read_line(
-                liner::Prompt::from("\x1B[1mredox login:\x1B[0m "),
+                liner::Prompt::from("\x1B[1meos login:\x1B[0m "),
                 None,
                 &mut liner::BasicCompleter::new(Vec::<String>::new()),
             )
-- 
2.53.0

